### PR TITLE
README back end API

### PR DESCRIPTION
**Pull request by dianemorrison**

### CODE REVIEW

**Review by cormac-larkin-kainos — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -6,8 +6,53 @@ How to start the floorIsJava application
...
+2. To see your application's Swagger UI http://localhost:8080/swagger
 
-Health Check
+Database
 ---
+Create an empty database: 
 
-To see your applications health enter url `http://localhost:8081/healthcheck`
+Run the following command from the base of your cloned directory to create the required database structure:
+`mysql --host=<localhost> --user=<your_username> --password=<your_password> <your_database_name> < employeesdb.sql`
```
Comment (shaunganley) [inline README.md]: do you have a sql file called employeesdb.sql? if not this line won't work

File: README.md [hunk 2]
```diff
@@ -6,8 +6,53 @@ How to start the floorIsJava application
...
+NOTE: Integration tests require connection to the database which may require VPN
+
+Build and run the service through docker
+
+You can build the api in a number of ways using docker and integrate it with a database, these are listed below: pre-requisite = docker and docker compose are installed in your local system.
+
+Ensure the environment variables are correct for your chosen db or enter these as additional arguments on the docker build command.
+
+
+Run "docker build <service name given + optional tag> ." from your src directory. This will read from your docker file, build the environment required for the image, build your service and create the image locally.
```
Comment (shaunganley) [inline README.md]: you also need to provide params when doing docker build

**Review by shaunganley — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (dianemorrison) [reply README.md]: reviewed and updated

File: README.md (hunk 2 above)
Comment (dianemorrison) [reply README.md]: added now

**Review by Thomas-Kainos — APPROVED** [review]

**Review by shaunganley — APPROVED** [review]

**Pull request merged**